draft-ietf-quic-transport.md

```diff
@@ -1109,7 +1109,7 @@ Stateless Reset do not contain frames.
 |                          Frame N (*)                        ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ~~~
-{: #packet-frames title="Contents of Protected Payload"}
+{: #packet-frames title="QUIC Payload"}
 
 Protected payloads MUST contain at least one frame, and MAY contain multiple
```

Comment:
You removed the term "protected payload" from the title of the figure, but not from the following paragraph.